src/components/content/order/scale/Scale.tsx

```diff
@@ -123,6 +134,37 @@ export const Scale = ({
         setScaleWarning('');
     };
 
+    const getServicePriceQuery = useGetServicePricesQuery(
+        currentSelectedService.serviceTemplateId ?? '',
+        currentSelectedService.deployRequest.region.name,
+        currentSelectedService.deployRequest.billingMode
+    );
+
+    const getFlavorWithPricesList = () => {
```

Comment:
this code is duplicated. 
We can directly do the processing as part of the useGetServicePricesQuery hook as well and directly use it from all necessary components.